source/Playnite/ViewModels/MainViewModelBase.cs

```diff
@@ -640,18 +640,42 @@ private async Task UpdateLibraryData(Func<CancellationToken, List<Game>> updateA
                         Logger.Info($"Downloading metadata for {addedGames.Count} new games.");
                         ProgressValue = 0;
                         ProgressTotal = addedGames.Count;
-                        ProgressStatus = Resources.GetString(LOC.ProgressMetadata);
+                        string progressBaseStr = ProgressStatus = Resources.GetString(LOC.ProgressMetadata);

```

Comment:
I don't understand why is this done this way now.

Reply:
Do you mean the string x = y = z notation, or the variable to save on Resources.GetString calls? If it's the latter I guess that might be premature optimization.